cockroachcloud/migrations-page.md

```diff
@@ -14,16 +14,28 @@ The **Migrations** page on the {{ site.data.products.db }} Console features a **
 - Create a new {{ site.data.products.serverless }} database that uses the converted schema. You specify the target database and database owner when [finalizing the schema](#finalize-the-schema). {% include cockroachcloud/migration/sct-self-hosted.md %}
 
 {{site.data.alerts.callout_info}}
-On the **Migrations** page, a *migration* refers to converting a schema for use with CockroachDB and creating a new database that uses the schema. It does not include moving data to the new database. For details on all steps required to complete a database migration, see [Migrate Your Database to CockroachDB](../{{version_prefix}}migration-overview.html).
+The **Migrations** page is used to convert a schema for use with CockroachDB and creating a new database that uses the schema. It does not include moving data to the new database. For details on all steps required to complete a database migration, see [Migrate Your Database to CockroachDB](../{{version_prefix}}migration-overview.html).
 {{site.data.alerts.end}}
 
 To view this page, select a cluster from the [**Clusters** page](cluster-management.html#view-clusters-page), and click **Migration** in the **Data** section of the left side navigation.
 
-## Upload a SQL dump
+## Schemas table
```

Comment:
I don't quite get the structure of this document. Won't most people be coming to this doc not having used the tool? Shouldn't we open with how to get started before showing the schemas table output? Or at least include a banner on to get started click here?

Reply:
This was originally conceived as a reference-style doc for the Migrations page. However, I agree that a task-based structure is more useful, so I've restructured accordingly. I think it's much better now!